exchanges/huobi/huobi_websocket.go

```diff
@@ -810,3 +801,31 @@ func (h *HUOBI) wsGetOrderDetails(ctx context.Context, orderID string) (*WsAuthe
 	}
 	return &response, nil
 }
+
+// channelName converts global channel Names used in config of channel input into exchange channel names
+// returns the name unchanged if no match is found
+func channelName(s *subscription.Subscription, p currency.Pair) string {
+	if n, ok := subscriptionNames[s.Channel]; ok {
+		return fmt.Sprintf(n, p)
+	}
+	if s.Authenticated {
+		panic(fmt.Errorf("%w: Private endpoints not currently supported", common.ErrNotYetImplemented))
+	}
+	panic(subscription.ErrPrivateChannelName)
```

Comment:
I'm fine with the error text, but `ErrPrivateChannelName` implies that it could only be an auth string that gets here, even with the Authenticated check above. It could be any garbage channel here eg `lkasjhdfalk` in my config, not necessarily private.
```[ERROR] | WEBSOCKET | 05/11/2024 10:19:29 | Huobi websocket: subscription failure, LOLBUTTS spot : template: master.tmpl:5:7: executing "master.tmpl" at <channelName $.S $p>: error calling channelName: must use standard channel name constants```

Reply:
Fixed e9f67fe99